### PR TITLE
Abble iCat (iCat)

### PR DESCRIPTION
**Pull request by RRWeb3**
Add Abble iCat, iCat
https://solscan.io/token/DppU9B7tPm9cFi5Kje1zCZU87w3xwftrH2vqiy94K5Ad

# Validate iCat (https://solscan.io/token/DppU9B7tPm9cFi5Kje1zCZU87w3xwftrH2vqiy94K5Ad)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://x.com/Abble_iCat/status/1807830613296402508
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/YOUR_ID_HERE

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Comment by RRWeb3** [comment]
Hello everyone! We need your support to get Abble iCat verified on Jupiter Exchange. Your active participation is crucial. Let's make it happen together!

**Review by RRWeb3 — COMMENTED** [review]
ok

**Review by RomanFoomi — COMMENTED** [review]
It's good

**Review by RomanFoomi — COMMENTED** [review]
Ok

**Review by RomanFoomi — COMMENTED** [review]
I didn't notice the checkbox)))

File: validated-tokens.csv
Comment (RomanFoomi) [inline validated-tokens.csv:1]: its my 4th coment)

**Review by RomanFoomi — COMMENTED** [review]
I watched

**Review by RomanFoomi — COMMENTED** [review]
Wht

**Comment by Jymfletzfur** [comment]
insufficient mcap

**Comment by RRWeb3** [comment]
> insufficient mcap



> insufficient mcap

Please provide us with the token evaluation criteria, so next time we can independently assess the likelihood of approval and ensure greater success.

**Pull request closed without merging**